README.md

```diff
@@ -575,6 +575,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 * [**Aggregator**](https://github.com/tughi/aggregator-android) <sup>**[[F-Droid](https://f-droid.org/app/com.tughi.aggregator)]**</sup>
 * [**Feeder**](https://gitlab.com/spacecowboy/Feeder) <sup>**[[F-Droid](https://f-droid.org/app/com.nononsenseapps.feeder)]**</sup>
+* [**HandyNewsReader**](https://github.com/yanus171/Handy-News-Reader) <sup>**[[F-Droid](https://f-droid.org/packages/ru.yanus171.feedexfork/)]**</sup>
```

Comment:
Minor correction, please use the [`/app/`](https://github.com/offa/android-foss/blob/master/CONTRIBUTING.md#f-droid-links) URL:
```suggestion
* [**HandyNewsReader**](https://github.com/yanus171/Handy-News-Reader) <sup>**[[F-Droid](https://f-droid.org/app/ru.yanus171.feedexfork)]**</sup>
```